kars/0003-gpu-sharing/README.md

```diff
@@ -0,0 +1,40 @@
+# KAR-0003: GPU Sharing
+
+## Description
+
+If accelerator supports GPU sharing, provide well-defined mechanisms for GPU sharing to improve utilization for workloads that do not require a full dedicated GPU. If hardware-level partitioning is supported, these fractional GPU resources should be exposed as distinct schedulable resources via DRA once it’s supported. If software-based sharing (e.g. time-slicing) is supported, device plugin should be used to allow oversubscription of GPUs, with a forward-looking path to DRA.
```

Comment:
Is a cloud platform compliant only if it supports **every available** GPU sharing strategy in a managed manner?